registrations/tests/management/test_delete_signups_with_expired_payments.py

```diff
@@ -0,0 +1,102 @@
+from datetime import timedelta
+
+import pytest
+from django.core.management import call_command
+from django.utils.timezone import localtime
+
+from registrations.models import SignUp, SignUpGroup, SignUpPayment
+from registrations.tests.factories import SignUpGroupFactory, SignUpPaymentFactory
+
+
+@pytest.mark.parametrize(
+    "threshold_days,expected_remaining_signups_count,expected_remaining_groups_count",
+    [
+        ("default", 5, 0),
+        (7, 3, 0),
+        (30, 6, 1),
+    ],
+)
+@pytest.mark.django_db
+def test_delete_signups_with_expired_payments(
```

Comment:
This test seems to wobble 1:1 with the underlying command so it passes DST transitions (checked because there is no freezetime). Anyway, that's :ok:

Reply:
Added freezetime to the test to make it more stable.